packages/eventual-send/src/handled-promise.js

```diff
@@ -477,7 +478,11 @@ export const makeHandledPromise = () => {
     applyMethodSendOnly: makeForwarder('applyMethodSendOnly', localApplyMethod),
   };
 
-  handle = (p, operation, opArgs, ...dispatchArgs) => {
+  handle = (...handleArgs) => {
+    // We're in SES mode, so we should harden.
+    harden(handleArgs);
```

Comment:
I thought we wanted to avoid implicitly hardening values, and consider it an error if the argument wasn't hardened already?

Reply:
@michaelfig and I had a long discussion in which he convinced me of the opposite. @michaelfig captured the rationale at https://github.com/Agoric/agoric-sdk/issues/4362

We should discuss.